docs/superpowers/plans/2026-07-01-edge-migration-phase2-runtime-parity-guard.md

```
@@ -0,0 +1,262 @@
+  for (const p of problems) console.error(`  - ${p}`);
+  process.exit(1);
+}
+console.log(`verify-runtime-schema-parity: OK (${FLOWS.length} flows: devices CHECK + trigger parity)`);
+process.exit(0);
+```
+
+- [ ] **Step 2: Run it — it must FAIL on the current regression**
+
+Run: `node scripts/verify-runtime-schema-parity.js`
+Expected: FAIL with **exactly** the CHECK problem for both profiles:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)